Samples/PublicSamples/EchoBot/ADF/main.ps1

```diff
@@ -0,0 +1,32 @@
+param (
+    [parameter(mandatory)]
+    [string]$OrgName,
+
+    [string]$Location = 'centralus',
+
+    [switch]$FullUpload,
+
+    [switch]$PackageDSC
+)
+
+$base = $PSScriptRoot
+$LocationLookup = Get-Content -Path $base/bicep/global/region.json | ConvertFrom-Json
+$Prefix = $LocationLookup.$Location.Prefix
+Write-Warning -Message "Prefix [$Prefix] Org [$OrgName] App [BOT] Environment [D1]"
+[String]$SAName = "${Prefix}${OrgName}botd1saglobal".tolower()
```

Comment:
That is the entry point. Otherwise you will need to request more parameters from the user. I updated it to have the app and environment variables to build the name exactly like the other files.